source/settings/command-line-options.txt

```diff
@@ -96,6 +103,10 @@ value in the :guilabel:`Settings` panel.
      - Enable :ref:`automatic updates <enable-automatic-updates>`. Use 
        ``--no-autoUpdates`` to disable automatic updates. 
 
+   * - .. option:: --browserCommandForOIDCAuth
+     - Specify a shell command to start the browser for OIDC authentication 
```

Comment:
[question] Should this be "specifies" to be consistent with the other options?

```suggestion
     - Specifies a shell command to start the browser for OIDC authentication 
```

Reply:
Updated.